internal/config/config.go

```diff
@@ -152,14 +197,30 @@ var regionalConfig = map[string]RegionalConfig{
 		},
 		Locale:       "en_gb",
 		NvidiaLocale: "en-gb",
-		Currency:     "EUR",
+		Currency:     "CZK",
```

Comment:
Have you actually tested these? Like browsed to the URLs and tried the add to checkout functionality locally to ensure they work? Before we make changes we need to validate these locally.

Reply:
yes

Reply:
https://api-prod.nvidia.com/direct-sales-shop/DR/products/en_gb/CZK/5394902800